spec/test.helper.ts

```diff
@@ -34,7 +34,9 @@ export class TestHelper {
                 return;
             }
             const entity = table.target as typeof BaseEntity;
-
+            if (!entity.query) {
+                return;
+            }
```

Comment:
ViewEntity does not have method `query`.
Instead, I registered the actual entity too for TypeORM in test code, so that it is used for deletion.